doc/code_style/mom.cpp

```diff
@@ -59,34 +59,32 @@
  * Will be parsed by Doxygen.
  */
 struct another_stupid_type {
-	rofl_t blergh_gnampf;  //!< variables and functions in lowercase
-	                       //   with _ as word separator.
-	epictype42_t løl;      //!< no question, we use UTF-8 encoding.
-	ub0rtype røfëlkøptør;  //!< proper documentation is Doxygen parsable.
+	rofl_t blergh_gnampf;     //!< variables and functions in lowercase
+	                          //   with _ as word separator.
+	epictype42_t løl;         //!< no question, we use UTF-8 encoding.
+	ub0rtype     røfëlkøptør; //!< proper documentation is Doxygen parsable.
 };
 
 
-
 /**
  * Yet another Valve game. Go on, nothing to see here.
  */
 class HalfLife3 : public ValveGame {
-// vvv The access modifier is at the same indent level as the class definition:
+	// vvv The access modifier is at the same indent level as the class definition:
 public:
-	void release_game(const ValveTeam &team,
+	void release_game(const ValveTeam& team,
 	                  uint128_t        passed_time,
-	                  const Engine    *game_engine,
-	                  Internet        *community,
-	                  bool             keep_secret=true) {
+	                  const Engine*    game_engine,
+	                  Internet*        community,
+	                  bool             keep_secret = true) {
 
 		// ^ insert a empty line here to make it more readable.
 
 		// http://www.abstrusegoose.com/440
-		auto gaben = team.get_human("gaben");
+		auto gaben       = team.get_human("gaben");
 		auto min_quality = gaben.get_quality_requirement(Engine::type);
 
-		if (not keep_secret
-		    and (this->story->awesomeness > 9000)
+		if (not keep_secret and (this->story->awesomeness > 9000)
```

Comment:
both are ok, but this piece of code should show how we would wrap boolean expressions

Reply:
Understood, needed to align that manually because the columnlimit of 100 characters was not reached.